PositionalGuide/PositionalGuide.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<Product>PositionalGuide</Product>
-		<Version>4.1.0</Version>
+		<Version>4.1.1</Version>
```

Comment:
This should be version `4.2.0` for a new feature, since I try to adhere to [semantic versioning](https://semver.org/) as best I can on a project that doesn't really have an API.